admin/release.py

```diff
@@ -448,16 +470,89 @@ def upload_rpms(scratch_directory, target_bucket, version, build_server):
         )
 
 
-def publish_rpms_main(args, base_path, top_level):
+def create_pip_index(scratch_directory, packages):
     """
-    The ClusterHQ yum repository contains packages for Flocker, as well as the
-    dependencies which aren't available in Fedora 20 or CentOS 7. It is
-    currently hosted on Amazon S3. When doing a release, we want to add the
-    new Flocker packages, while preserving the existing packages in the
-    repository. To do this, we download the current repository, add the new
-    package, update the metadata, and then upload the repository.
+    Create an index file for pip.
 
-    :param list args: The arguments passed to the script.
+    :param FilePath scratch_directory: Temporary directory to create index in.
+    :param list packages: List of bytes, filenames of packages to be in the
+        index.
+    """
+    index_file = scratch_directory.child('index')
+    with index_file.open('w') as f:
+        f.write('<!--This is an index for pip-->\n')
```

Comment:
No reason for this to be a HTML comment. If somebody navigates to the page, they'll see this.